### PR TITLE
fix: show current context in quick pick

### PR DESCRIPTION
**Pull request by deboer-tim**
### What does this PR do?

Even when the quick pick is fixed to show which items are pre-selected (issue #2754 item 2) it is not really clear to the user why one row is highlighted. This adds the text " (current)" to the current Kube context to make this obvious.

### Screenshot/screencast of this PR
Screenshot 2023-06-19 at 9 01 59 AM

### What issues does this PR fix or reference?

N/A

### How to test this PR?

Click on the context status bar to see the quick pick.

### CODE REVIEW

**Comment by benoitf** [comment]
shouldn't we fix the quickpick component instead ?

**Comment by deboer-tim** [comment]
We should fix the quick pick too, but to me it's still not obvious _why_ one item is selected by default. e.g. in the image if kind-kind-cluster-5 was selected I still need to think for a second to realize that's because it's the current context. I'm not sure how else to show this, especially for new users.

**Comment by benoitf** [comment]
@deboer-tim should we put the current item as the first item of the list ?

**Comment by benoitf** [comment]
I mean, the extension could always set as current value the one being selected (independent of the quickpick)

File: extensions/kube-context/src/extension.ts [hunk 1]
```diff
@@ -110,7 +110,7 @@ export async function updateContext(
 
   quickPicks = contexts.map(context => {
     return {
-      label: context.name,
+      label: context.name === currentContext ? context.name + ' (current)' : context.name,
```
Comment (benoitf) [inline extensions/kube-context/src/extension.ts]:
```suggestion
      label: context.name === currentContext ? `${context.name} (current)` : context.name,
```

**Comment by jeffmaury** [comment]
I think the current context should be the one selected by default and VSCode OpenShift uses another styling:

image

**Comment by deboer-tim** [comment]
Quickpicks can have a description?? 😀

Switched to use the same styling, using a current context description is better than changing the label:
Screenshot 2023-06-19 at 11 44 07 AM

**Review by benoitf — APPROVED** [review]

**Pull request merged**